src/mono/netcore/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -139,6 +139,7 @@
     <FeaturePortableTimer>true</FeaturePortableTimer>
     <FeaturePortableThreadPool>true</FeaturePortableThreadPool>
     <FeaturePerfTracing>true</FeaturePerfTracing>
+    <FeatureAppX Condition="'$(TargetsWindows)' == 'true'">true</FeatureAppX>
```

Comment:
I do not think we should be defining this for Mono. It would be better to either add stubs for whatever is missing.

Reply:
K. I think I already mostly stubbed this out so removing shouldn't be a big change.

Reply:
I am not sure we have prior art here but it seems to me to be more logical to have the fully stubbed versions (PNSE) of public types in shared location